pkg/resource/meta/resource_definition.go

```diff
@@ -31,12 +31,16 @@ type PrintColumn struct {
 }
 
 // ResourceDefinitionSpec provides ResourceDefinition definition.
-type ResourceDefinitionSpec struct {
+type ResourceDefinitionSpec struct { //nolint:govet
 	Type             resource.Type      `yaml:"type"`
 	DisplayType      string             `yaml:"displayType"`
 	DefaultNamespace resource.Namespace `yaml:"defaultNamespace"`
 	Aliases          []resource.Type    `yaml:"aliases"`
 	PrintColumns     []PrintColumn      `yaml:"printColumns"`
+
+	// Sensitivity indicates how secret resource of this type is.
+	// The empty value represents a non-sensitive resource.
```

Comment:
The alternative is to make the zero value mean "unspecified," but I would rather not introduce undefined behavior.